Best practices/Enzyme.md

```diff
@@ -78,3 +78,19 @@ expect(myComponent.find(ChildComponent)).toHaveLength(1);
 // over:
 expect(myComponent.find(ChildComponent).exists()).toBe(true);
 ```
+
+## Event simulations
+
+* When triggering props on a composite component, use `trigger`; when doing it for raw DOM elements, use `simulate`.
+
+```js
+const myComponent = mount(<MyComponent />);
+
+trigger(myComponent, 'onClick');
+```
+
+```js
+const myComponent = mount(<MyComponent />);
+
+myComponent.simulate('click');
+```
```

Comment:
Needs a newline, might be worth having the counter examples for each of these too.